File: spring-ai-a2a-examples/java/v1/host-agent/src/main/java/io/github/timo/a2a/spring/ai/agent/example/HostAgentApplication.java
package io.github.timo.a2a.spring.ai.agent.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class HostAgentApplication {

    public static void main(String[] args) {
        SpringApplication.run(HostAgentApplication.class, args);
    }
}

File: spring-ai-a2a-examples/kotlin/v1/host-agent/src/main/kotlin/io/github/timo/a2a/spring/ai/agent/example/HostAgentApplication.kt
package io.github.timo.a2a.spring.ai.agent.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class HostAgentApplication

fun main(args: Array<String>) {
    runApplication<HostAgentApplication>(*args)
}
